package.json

```diff
@@ -65,30 +67,34 @@
   },
   "homepage": "https://github.com/react-native-community/datetimepicker#readme",
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/runtime": "^7.17.8",
-    "@react-native-community/eslint-config": "^2.0.0",
+    "@babel/core": "^7.19.0",
+    "@babel/preset-env": "^7.19.0",
```

Comment:
I just added this because it includes a few dependencies that react-native needs, @babel/plugin-transform-regenerator, @babel/plugin-transform-for-of, ...